src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -165,15 +165,15 @@ object SbtBundle extends AutoPlugin {
     }.value,
     NativePackagerKeys.stage in BundleConfiguration := Def.taskDyn {
       Def.task {
-        stageConfiguration
+        stageConfiguration()
       }.value
     }.value,
     NativePackagerKeys.stagingDirectory in Bundle := (target in Bundle).value / "stage",
     NativePackagerKeys.stagingDirectory in BundleConfiguration := (target in BundleConfiguration).value / "stage",
     target in Bundle := target.value / "bundle",
     target in BundleConfiguration := target.value / "bundle-configuration",
-    sourceDirectory in BundleConfiguration := sourceDirectory.value / "bundle-configuration",
-    configurationName := "default "
+    sourceDirectory in BundleConfiguration := (sourceDirectory in Universal).value.getParentFile / "bundle-configuration",
```

Comment:
This is the major change - files will reside in the idiomatic place for Play apps now.